.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Default owner(s) of all files in this repository
-* @cmsirbu @snaselj @whitej6
+* @cmsirbu @gsnider2195 @smk4664
```

Comment:
Do we need an app maintainers team in github to add to codeowners here or is this ok?

Reply:
[@maintain-nautobot-apps](https://github.com/orgs/nautobot/teams/maintain-nautobot-apps)